source/getting_started/aiot_intelligent_edge/coco_for_applications/analytics_client_applications/browser_js_analytics_client_applications/quick_start_guide_browser_js_analytics_client_apps.rst

```diff
@@ -3,12 +3,27 @@
 Quick Start Guide
 =================
 
-.. figure:: /_static/coming-soon.png
-   :align: center
+.. sectionauthor:: Deekshith
 
-   **Coming Soon**
+.. Suggestions from Ashish - Steps to Sync Sources, Compilation, and Running Successfully. Also mentioning how Auto-Magically things connect to the COCO App
 
-.. sectionauthor:: Deekshith
+Application Build Process
```

Comment:
Please use the formatting/style and steps similar to the COCO Device SDK Quick Start Guide

Reply:
handled